### PR TITLE
create Jupyterhub k8s network tools package

### PR DESCRIPTION
**Pull request by sanghanan**
Fixes:

Related: https://github.com/chainguard-dev/image-requests/issues/3789

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)
- [ ] If non-streamed package names no longer built, open PR to withdraw them (see [WITHDRAWING PACKAGES](https://github.com/wolfi-dev/os/blob/main/WITHDRAWING_PACKAGES.md))

#### For package updates (renames) in the base images

When updating packages part of base images (i.e. cgr.dev/chainguard/wolfi-base or ghcr.io/wolfi-dev/sdk)
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk images successfully build
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk contain no obsolete (no longer built) packages
- [ ] Upon launch, does `apk upgrade --latest` successfully upgrades packages or performs no actions

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: jupyterhub-k8s-network-tools.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  name: jupyterhub-k8s-network-tools
+  version: 3.3.8
+  epoch: 0
+  description: Network diagnostic tools for use within a JupyterHub Kubernetes cluster
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - iptables
+
```
Comment (mamccorm) [inline jupyterhub-k8s-network-tools.yaml:11]: Hey @sanghanan, is this commit missing some changes? This just looks like a package which includes iptables, runs a test, then it packages it up without any changes?

File: jupyterhub-k8s-network-tools.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+test:
+  pipeline:
+    - name: Test iptables
+      runs: |
+        iptables --version
+
+update:
+  enabled: true
+  github:
```
Comment (mamccorm) [inline jupyterhub-k8s-network-tools.yaml:30]: We're polling Git for updates, but theres no Git clone happening in this melange?

File: jupyterhub-k8s-network-tools.yaml (hunk 1 above)
Comment (sanghanan) [reply jupyterhub-k8s-network-tools.yaml:11]: Hey Mark! This is a weird package
Here's the thread for context - https://github.com/chainguard-dev/image-requests/issues/3789
We're essentially just packaging up wolfi base with iptables
If you see the [dockerfile](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/main/images/network-tools/Dockerfile), it contains just 2 lines: 
```dockerfile
FROM alpine:3.18
RUN apk add --no-cache iptables
```
If there's a better way to do this, I'd love to know!

File: jupyterhub-k8s-network-tools.yaml (hunk 2 above)
Comment (sanghanan) [reply jupyterhub-k8s-network-tools.yaml:30]: We're polling github, just to tag the package correctly
There is nothing else to be used from the repository at all

File: jupyterhub-k8s-network-tools.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package:
```
Comment (mamccorm) [inline jupyterhub-k8s-network-tools.yaml:1]:
```suggestion
At the time of writing, that image only contains iptables. We also need the
package to take it's versioning from the upstream repo. Essentially this is just
a wolfi-base image with iptables, that takes it's versioning from the upstream
image repo (Until uch times that upstream adds additional tool in future). See:
 - https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/main/images/network-tools/Dockerfile
```

File: jupyterhub-k8s-network-tools.yaml (hunk 1 above)
Comment (mamccorm) [reply jupyterhub-k8s-network-tools.yaml:11]: Thanks @sanghanan that helps. Added an inline suggestion above re: code comments

File: jupyterhub-k8s-network-tools.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (sanghanan) [reply jupyterhub-k8s-network-tools.yaml]: Thanks, @mamccorm

File: jupyterhub-k8s-network-tools.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+Creates a package which represents the following upstream image:
```
Comment (jamonation) [inline jupyterhub-k8s-network-tools.yaml:1]: Something got messed up here - I think some feedback ended up being a code suggestion. Once this is fixed it should pass CI

Comment (sanghanan) [reply jupyterhub-k8s-network-tools.yaml]: This should work 🤞🏽

**Review by jamonation — APPROVED** [review]

**Pull request merged**